cache/client.go

```diff
@@ -39,6 +41,14 @@ type clientMetrics struct {
 func newClientMetrics(reg prometheus.Registerer) *clientMetrics {
 	cm := &clientMetrics{}
 
+	cm.requests = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+		Name: "requests_total",
+		Help: "Total number of items requests to cache.",
```

Comment:
Note that we don't set a constant label with the name of the cache anymore: Both `MemcachedClient` and `RedisClient` already wrap the registries they use in their respective constructors with the cache name label.

Reply:
maybe just a short comment or docstring about this - something like "registries passed in should be wrapped with an implementation-specific value for the `labelCacheBackend` label"

Reply:
Fixed in 739f646a4ebf59eff78a659aae90440c31732a59